### DISCUSSION TITLE
How can I add POCKETSVG_DISABLE_FILEWATCH using swift?

### DISCUSSION THREAD

**Discussion by fluidvm** [pocketsvg/PocketSVG#209] [Q&A]
I wish there was a simple true/false property, but in the code I've found this: POCKETSVG_DISABLE_FILEWATCH

Where to add it using swift and package manager?

**Comment by FlashTang** [comment]
For anyone who don't know how to add this
1) search "CC_PREPROCESSOR_DEFINITIONS" in the PokectSVG target (not in your main project but Pods project)
2) add "POCKETSVG_DISABLE_FILEWATCH=1" to debug and release  (without quotation mark "" )
build your project , and then you can see the  codes under #if defined(DEBUG) && !defined(POCKETSVG_DISABLE_FILEWATCH)  are grayed out
